pkg/controllers/status/cluster_status_controller.go

```diff
@@ -256,22 +269,22 @@ func (c *ClusterStatusController) updateStatusIfNeeded(cluster *clusterv1alpha1.
 	return controllerruntime.Result{RequeueAfter: c.ClusterStatusUpdateFrequency.Duration}, nil
 }
 
+func (c *ClusterStatusController) initializeGenericInformerManagerForCluster(clusterClient *util.ClusterClient) {
+	if c.GenericInformerManager.IsManagerExist(clusterClient.ClusterName) {
+		return
+	}
+
+	dynamicClient, err := c.ClusterDynamicClientSetFunc(clusterClient.ClusterName, c.Client)
+	if err != nil {
+		klog.Errorf("Failed to build dynamic cluster client for cluster %s.", clusterClient.ClusterName)
+		return
```

Comment:
Should we return err? (keep the behavior as before)

Reply:
I think just one log is enough. 
Even in case of error, what else can the cluster status controller do except log again the error? 
The error shouldn't block the normal process.